modules/help_editor/php/helpfile.class.inc

```diff
@@ -87,7 +87,7 @@ class HelpFile
         $DB =& \Database::singleton();
 
         // insert a help file
-        $success = $DB->insert('help', $set);
+        $success = $DB->unsafeinsert('help', $set);
```

Comment:
@driusan we decided on changing this to unsafeinsert back in december. and now i'm a little hesitant. does this still make sense to you?

Reply:
I've always been a little hestitant, but I don't see what other option there is to avoid the mangling.. what would you propose instead?

Reply:
i don't have anything in mind, just wanted to see where you were at!